Casks/l/little-snitch.rb

```diff
@@ -8,8 +8,10 @@
   homepage "https://www.obdev.at/products/littlesnitch/index.html"
 
   livecheck do
-    url "https://www.obdev.at/products/littlesnitch/download.html"
-    regex(%r{href=.*?/LittleSnitch[._-]v?(\d+(?:\.\d+)+)\.dmg}i)
+    url "https://sw-update.obdev.at/update-feeds/littlesnitch#{version.major}.plist"
+    strategy :xml do |xml|
+      xml.get_elements("//key[text()='BundleShortVersionString']").map { |item| item.next_element&.text&.strip }
```

Comment:
```suggestion
    regex(/LittleSnitch[._-]v?(\d+(?:\.\d+)+)\.dmg/)
    strategy :xml do |xml|
      xml.get_elements("/plist/array/dict")&.map do |item|
        download_url = item.elements["//key[text()='DownloadURL']"]&.next_element&.text&.strip
        next if download_url.blank?

        download_url[regex, 1]
      end
```

The plist contains both a stable and nightly version and we only want to return the stable version(s), so this ends up being a little more involved. The "final" and "nightly" versions are both 5.7.4 at the moment but the current approach in the PR would become a problem if/when they diverge.

We could filter based on the `ReleaseLifecycle` value (e.g., `final` for stable, `nightly` for unstable) but I've opted to match stable `DownloadURL` filenames (e.g., `LittleSnitch-5.7.4.dmg` instead of `LittleSnitch-5.7.4-nightly-(6300).dmg`), so we're not reliant on specific `ReleaseLifecycle` values.

Reply:
Thanks @samford.